docs/src/modules/components/MarkdownDocs.js

```diff
@@ -178,7 +178,7 @@ function MarkdownDocs(props) {
             </Portal>
           )}
 
-          <AppContent disableToc={disableToc} className={classes.root}>
+          <AppContent disableToc={disableToc}>
```

Comment:
A dead class name found with the new warning.

Reply:
If you're concerned with using missing classes then using types is the better approach.